Mods/vcmi/config/vcmi/english.json

```diff
@@ -72,7 +73,6 @@
 	"vcmi.lobby.noUnderground" : "no underground",
 	"vcmi.lobby.sortDate" : "Sorts maps by change date",
 
-	"vcmi.client.errors.invalidMap" : "{Invalid map or campaign}\n\nFailed to start game! Selected map or campaign might be invalid or corrupted. Reason:\n%s",
```

Comment:
why was this removed?

Reply:
Was an mistake. See the full change history...